ledger-core/v2/Makefile.testing

```diff
@@ -31,4 +31,33 @@ test_func_race: ## run functest 10 times with -race flag
 test_slow: ## run tests with slowtest tag
 	$(GOTEST) -count=1 $(TEST_ARGS) -tags slowtest $(PACKAGES)
 
+.PHONY: kube_start_net
+kube_start_net: ## run insolar net in local kubernetes cluster
+	./deploy/kube/run.sh
+
+.PHONY: kube_stop_net
+kube_stop_net: ## stop insolar net in local kubernetes cluster
+	./deploy/kube/stop.sh
+
+.PHONY: kube_collect_artifacts
+kube_collect_artifacts: ## stop insolar net in local kubernetes cluster
+	./deploy/kube/collect_artifacts.sh
```

Comment:
Some of our developers use Windows. Somehow I doubt these .sh files will work there. Please use Python.

Reply:
Sorry, don't have opportunity to test on win docker desktop, but I tested it on docker desktop for mac and it works. There is a target in makefile "test_func_kubernetes", it will use your local cluster

Reply:
I suggest to open ticket for Windows support and implement it within that ticket

Reply:
We decided to make a subtask.